src/app/carbon-estimation-table/carbon-estimation-table.component.html

```diff
@@ -0,0 +1,72 @@
+<!-- eslint-disable @angular-eslint/template/interactive-supports-focus -->
+<table role="treegrid" aria-label="Estimation of emissions" class="tce-table-fixed tce-w-full">
+  <thead>
+    <tr>
+      <th>Category</th>
+      <th class="tce-w-40">Emissions</th>
+    </tr>
+  </thead>
+  <tbody
+    (keydown.home)="homeEndKeyBoardEvent($event, true)"
+    (keydown.end)="homeEndKeyBoardEvent($event, false)"
+    (keydown.control.home)="homeEndKeyBoardEvent($event, true)"
+    (keydown.control.end)="homeEndKeyBoardEvent($event, false)"
+    (keydown.arrowup)="arrowKeyBoardEvent($event, 'up')"
+    (keydown.arrowdown)="arrowKeyBoardEvent($event, 'down')">
```

Comment:
Is it the fact that sections of the table collapse that necessitates custom handlers for keyboard interaction? Whilst I think the table looks great and works fine for mouse users this amount of programmatic handling of keystrokes and setting the tabindex looks like a red flag to me in terms of accessibility in the sense that I think we'll probably have fallen into a lot of traps on the way. There's also quite a lot of code to grok and work on if things do need to be changed. 

I'm wondering if the sections should really be collapsible. It's quite a nice feature for mouse users but there isn't that much information in the table when expanded so I don't think collapsing the table makes a huge difference in terms of parsing the data. Given that one of the main reasons for adding the table was to make the data more accessible I would be tempted to remove the collapsing section feature if that's the reason for the complexity.